web/src/app/app.module.ts

```diff
@@ -85,6 +87,8 @@ import {PlatformModule} from '@angular/cdk/platform';
     FormsModule,
     HttpClientModule,
     MaterialModule,
+    MatDialogModule,
```

Comment:
remove MatDialogModule and MatSnackBarModule please; these two has been included in the material module 

https://github.com/MindongLab/yngdieng/blob/master/web/src/material/material.module.ts#L24